### PR TITLE
Adding support for AWS transport

### PR DESCRIPTION
**Pull request by brendangibat**

### CODE REVIEW

**Comment by analytically** [comment]
Hi, thanks for your contribution. Mind fixing the issues: https://travis-ci.org/sensu-plugins/sensu-plugins-elasticsearch/jobs/94043704, thanks!

**Comment by brendangibat** [comment]
Of course, I was just getting things started on the PR.

**Comment by sstarcher** [comment]
@brendangibat ping pong

**Comment by blakebarnett** [comment]
Are rubocop complaints the only thing preventing this from going in?

File: bin/check-es-query-count.rb [hunk 1]
```diff
@@ -170,36 +187,69 @@ class ESQueryCount < Sensu::Plugin::Check::CLI
          description: 'Invert thresholds',
          boolean: true
 
+  option :kibana_url,
+         long: '--kibana-url KIBANA_URL',
+         description: 'Kibana URL query prfix that will be in critical / warning response output.'
```
Comment (sstarcher) [inline bin/check-es-query-count.rb]: typo prfix

**Comment by sstarcher** [comment]
@blakebarnett that and the typo I believe are about it

**Comment by sstarcher** [comment]
m

**Comment by brendangibat** [comment]
ty

**Pull request closed without merging**